client/layout/universal-navbar-footer/index.tsx

```diff
@@ -142,6 +148,13 @@ const UniversalNavbarFooter = () => {
 										<span className="lp-link-chevron-external">{ translate( 'Policy' ) }</span>
 									</a>
 								</li>
+								{ shouldSeeDoNotSell && (
+									<li>
+										<button onClick={ openDialog }>
+											{ preventWidows( translate( 'Do Not Sell or Share My Data' ) ) }
```

Comment:
I find it odd that in other menu items here we use sentence case, but here, we use title case. Should we try to be consistent about it?

Reply:
Hmm, from what I see in the universal navbar footer, we are aligned with title case:
![image](https://user-images.githubusercontent.com/8419292/207375884-c13cceec-3dbb-48a2-9b57-9933490836ae.png)

e.g. "Built by WordPress.com", "Terms of Service". Or do I miss something?

Reply:
You're right - I must have been looking at different strings - disregard.